Source/SwiftLintFramework/Rules/Lint/PrivateUnitTestRule.swift

```diff
@@ -106,12 +106,48 @@ public struct PrivateUnitTestRule: SwiftSyntaxRule, ConfigurationProviderRule, C
                 private ↓func test4() {}
             }
             """)
+        ],
+        corrections: [
+            Example("""
+
+                private class Test: XCTestCase {}
+                """): Example("""
+
+                    class Test: XCTestCase {}
+                    """),
+            Example("""
+                class Test: XCTestCase {
+
+                    private func test1() {}
```

Comment:
Should these examples have violation markers?

Reply:
Yes. They now have.